Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,18 @@
+from fedora:32
+
+WORKDIR /project
+
+RUN dnf update -y;
+RUN mkdir /project/output
+RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac); \
+    export OS=$(uname | awk '{print tolower($0)}'); \
+    export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.4.0/; \
+    curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH};\
+    chmod +x operator-sdk_${OS}_${ARCH};\
+    mv operator-sdk_${OS}_${ARCH} operator-sdk;\
+    mv operator-sdk /usr/local/bin/;
+RUN dnf install -y git ansible wget python3-pip
+RUN git clone --branch v1.0.11 https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
```

Comment:
Do we want to have the branch (tag) of the operator-test-playbooks as an argument with the default value of `v1.0.11`?

Reply:
@dirgim same question as above.

Reply:
This might change less - but if we want to introduce a bugfix/update to any of the roles used by midstream, we will want to change it.
Also having the versions/releases set as arguments should hopefully make it more obvious that it needs/can be updated or modified accordingly.